core/aws_ddk_core/pipelines/state_machine_stage.py

```diff
@@ -1,154 +0,0 @@
-# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License").
-# You may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from abc import abstractmethod
-from typing import List, Optional
-
-from aws_cdk.aws_events import EventPattern, IRuleTarget
-from aws_cdk.aws_events_targets import SfnStateMachine
-from aws_cdk.aws_stepfunctions import IChainable, StateMachine, StateMachineType
-from aws_cdk.aws_stepfunctions_tasks import EventBridgePutEvents, EventBridgePutEventsEntry
-from aws_ddk_core.pipelines.stage import DataStage
-from aws_ddk_core.resources import StepFunctionsFactory
-from constructs import Construct
-
-
-class StateMachineStage(DataStage):
```

Comment:
Tbh I thought we'd leave this class even if we switch to events automatically published by sfn instead of publishing manually. It would still be useful, for example if you'd like to use `EXPRESS` workflow that doesn't publish any events. Are we ok to force all users to use `STANDARD` workflow only?

Reply:
Yeah the more I think about it the less I wish to encourage people to use `EXPRESS`. It seems ill-suited for D&A workloads and reserved for high concurrency, short duration workflows like fulfilling HTTP requests for a website.

But I am open to keeping this class if you wish. Just felt like tech debt for a narrow use case.